karabo_bridge/simulation.py

```diff
@@ -283,6 +287,16 @@ def start_gen(port, ser='msgpack', version='2.2', detector='AGIPD',
         Number of sources.
     datagen: string, optional
         Generator function used to generate detector data. Default is random.
+    data_like: string optional ['online', 'file']
+        Data array axes ordering for Mhz detector.
+        The data arrays's axes can have different ordering on online data. The
+        calibration processing orders axes as (fs, ss, pulses), whereas
+        data in files have (pulses, ss, fs).
+        This option allow to chose between 2 ordering:
+        - online: (modules, fs, ss, pulses)
+        - file: (pulses, modules, ss, fs)
+        Note that the real system can send data in both shape with a
+        performance penalty for the file-like array shape, default is online.
```

Comment:
Nitpick: this sounds like it's describing the default for the 'real' system. Maybe put it in a separate paragraph to be clear that we're describing the default for the simulator.